test/unit/MigrationActions.t.sol

```diff
@@ -14,55 +14,55 @@ import { MigrationActions } from "src/MigrationActions.sol";
 abstract contract MigrationActionsBase is Test {
 
     MockERC20   public dai;
-    MockERC20   public nst;
+    MockERC20   public usds;
     ERC4626Mock public sdai;
-    ERC4626Mock public snst;
+    ERC4626Mock public susds;
 
     VatMock  public vat;
     JoinMock public daiJoin;
-    JoinMock public nstJoin;
+    JoinMock public usdsJoin;
 
     MigrationActions public actions;
 
     address receiver = makeAddr("receiver");
 
     function setUp() public {
-        dai  = new MockERC20("DAI", "DAI", 18);
-        nst  = new MockERC20("NST", "NST", 18);
-        sdai = new ERC4626Mock(dai, "sDAI", "sDAI", 18);
-        snst = new ERC4626Mock(nst, "sNST", "sNST", 18);
+        dai   = new MockERC20("DAI", "DAI", 18);
+        usds  = new MockERC20("USDS", "USDS", 18);
```

Comment:
Align params

Reply:
were these aligned before? It's a bit awkward

Reply:
fixed